README.md

```diff
@@ -1,26 +1,40 @@
 # IWC - Intergalactic Workflow Commission
 
-## This group exists to create a registry of community workflows. 
-
-### Reproducibility is important. Our goals are to:
- - foster workflow use
- - incorporate versioning
- - capture more metadata: (names, versions, authors, use cases, etc.)
- - help scientists find workflows!
-
-### Methods Proposal
- - discover workflows
- - establish quality standards (version numbers, tools exist, author is set, training available, test exists ...)
- - automated testing of workflows utilizing [Planemo](https://github.com/galaxyproject/planemo) test
- - two layers of test data:
-   - basic functionality testing (quick to run, tools exist) 
-   - downsampled data (validity of workflow results)
-
-### Import process:
- - submit a pull request (PR) that adds a link to a GA file and metadata  
- - confirm that the GA and metadata conform to standards (linting criteria to be defined)
- - test the workflow 
- - if linting passes, tests pass, and human review passes, PR is merged and added to registry list
-
-### Meeting minutes:
- * [the notes](https://github.com/galaxyproject/iwc/issues/3) from the initial meeting at GCCBOSC2018 CoFest
+[![Galaxy Workflow Linting and Tests](https://github.com/galaxyproject/iwc/actions/workflows/workflow_test.yml/badge.svg?branch=main)](https://github.com/galaxyproject/iwc/actions/workflows/workflow_test.yml/badge.svg?branch=main)
+[![Gitter](https://badges.gitter.im/galaxyproject/iwc.svg)](https://gitter.im/galaxyproject/iwc?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+
+## The IWC maintains high-quality Galaxy Workflows
+
+Workflows are categorized in the [workflows](workflows/) directory, and listed in [Dockstore](https://dockstore.org/organizations/iwc) and [WorkflowHub](https://workflowhub.eu/search?utf8=%E2%9C%93&q=IWC#workflows).
+
+All workflows are reviewed and tested before publication and with every new Galaxy release.
+Deposited workflows follow [best practices](https://planemo.readthedocs.io/en/latest/best_practices_workflows.html) and are versioned using github releases.
+Workflows also contain important metadata, such as:
+
+- License
+- Author
+- Institutes
+
+Additionally the IWC will collect further best practices, tips and tricks, FAQs and assist the community in designing high-quality Galaxy workflows.
+
+## Importing Workflows into Galaxy
+
+To import IWC workflows into your Galaxy instance use the TRS workflow search in the Galaxy interface.
+Click on "Workflows", "Import" and click on the "search form" link.
+![Workflow Import GIF](static/iwc-import.gif)
+
+When importing a workflow form workflowhub.eu you currently need to search for `organization: "IWC Workflows"`.
```

Comment:
```suggestion
When importing a workflow form workflowhub.eu you currently need to search for `organization:"IWC Workflows"`.
```

a space between the key and the value seems to break things, too.

Reply:
Ugh ... we really need to have that search interface you suggested.

Reply:
Quick fix is https://github.com/galaxyproject/galaxy/pull/12197, then we also don't need to quotes anymore

Reply:
only for 21.05 I guess ;-)

Reply:
We can backport this a bit further when it has stabilised 😆